### DISCUSSION TITLE
How to Hide Debug Info In PIMCORE's Portal Page

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#16643] [Q&A / Support]
Hi All,

If production mode is enabled in PIMCore the datahub will not available in PIM admin page how to enable datahub in production mode at the same time the debug info will not be show in PXP of Pimcore, is there any solution is there for this.

**Comment by KasthuriAnandhan** [comment]
Is anybody can help me here?

**Comment by KasthuriAnandhan** [comment]
Hi @kingjia90 can you help me for this.

**Reply by kingjia90** [reply]
Sorry, but can't do it here, since it's Portal Engine related, please get in touch with the sales team to get know about the [SLA](https://pimcore.com/en/services/support)

**Comment by KasthuriAnandhan** [comment]
THANK YOU
